ccd-definition/CaseField.json

```diff
@@ -341,7 +341,15 @@
     "LiveFrom": "01/01/2017",
     "CaseTypeID": "CARE_SUPERVISION_EPO",
     "ID": "gateKeeperLabel",
```

Comment:
if it isn't too much change could we change the names of all of these to have lower case 'k'.
I know we might not be able to for `gateKeeperEmail` but the others are new/labels

Reply:
Agreed. Let's do this for email collection and the newly added hint label.